config.json

```diff
@@ -335,401 +344,426 @@
+        "name": "Triangle",
+        "uuid": "5c6bc8d2-2509-471a-993c-a5bf9d030ca7",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 3,
+        "topics": [
+          "booleans",
           "control_flow_if_else_statements",
-          "math",
-          "memory_management"
+          "structs"
         ]
       },
       {
-        "slug": "binary",
-        "name": "Binary",
-        "uuid": "8621177c-1068-4610-bf5b-26cf23340ae6",
+        "slug": "luhn",
+        "name": "Luhn",
+        "uuid": "8c0762e6-702e-4733-a0a1-65099e2a46f7",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 1,
+        "difficulty": 4,
         "topics": [
-          "control_flow_if_statements",
-          "control_flow_loops",
-          "math"
+          "algorithms",
+          "strings"
         ]
       },
       {
-        "slug": "queen-attack",
-        "name": "Queen Attack",
-        "uuid": "b0a152e9-5a45-41f9-bda0-427111d9a56c",
+        "slug": "allergies",
+        "name": "Allergies",
+        "uuid": "871f1c41-debd-4807-8ee5-bde54c3918f8",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 1,
+        "difficulty": 4,
         "topics": [
           "control_flow_if_statements",
-          "enums",
-          "functions",
+          "control_flow_loops",
           "structs"
         ]
       },
       {
-        "slug": "linked-list",
-        "name": "Linked List",
-        "uuid": "58107e5f-8091-4429-9130-13f2c7dac9a9",
+        "slug": "complex-numbers",
+        "name": "Complex Numbers",
+        "uuid": "472b1a6b-2b7a-4609-a97d-4c2b6c941a1f",
         "practices": [],
         "prerequisites": [],
         "difficulty": 4,
         "topics": [
-          "functions",
-          "lists",
-          "pointers",
+          "math",
+          "performance_optimizations",
           "structs"
         ]
       },
       {
-        "slug": "darts",
-        "name": "Darts",
-        "uuid": "a579e471-86f3-4f29-82ca-e984a2cc8891",
+        "slug": "sum-of-multiples",
+        "name": "Sum of Multiples",
+        "uuid": "06a3660f-16b0-4391-939e-a9786887b0b2",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 1,
+        "difficulty": 4,
         "topics": [
-          "control_flow_if_else_statements",
-          "logic",
+          "control_flow_case_statements",
+          "control_flow_loops",
           "math"
         ]
       },
       {
-        "slug": "resistor-color-duo",
-        "name": "Resistor Color Duo",
-        "uuid": "93271b2e-8ad9-4e7d-8732-5988140e8c00",
+        "slug": "clock",
+        "name": "Clock",
+        "uuid": "b868eeff-400c-4e46-8675-10d85c14b503",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 1,
+        "difficulty": 4,
         "topics": [
-          "arrays",
-          "enums"
+          "control_flow_if_statements",
+          "preprocessor_x_macros_in_test",
```

Comment:
I think there are no X-macros here.
Perhaps there was at some point in the past.
Should this topic be removed? If so, will it affect tje difficulty rating or ordering?

Reply:
Yep, I'll remove it. I don't think it makes much of a difference for difficulty. I rated it a 4 because it has string parsing and formatting, both of which are a bit of a pain for a C newbie.